### PR TITLE
Fix two minor issues in shell integration

### PR DESCRIPTION
**Pull request by grimreaper**
- speeling error in a comment
- extraneous "\" escape

### CODE REVIEW

File: source/shell_integration/bash [hunk 1]
```diff
@@ -567,7 +567,7 @@ function __iterm2_precmd () {
     fi
 
     # Get the value of the prompt prefix, which will change $?
-    \local iterm2_prompt_prefix_value="$(iterm2_prompt_prefix)"
+    local iterm2_prompt_prefix_value="$(iterm2_prompt_prefix)"
```
Comment (gnachman) [inline source/shell_integration/bash]: This is intentional. It means "use the real `local`, even if there is an alias with that name"

Comment (grimreaper) [reply source/shell_integration/bash]: ah. I usually use single quotes.

**Comment by gnachman** [comment]
Didn't know you could do that with single quotes! Thanks :)

**Comment by grimreaper** [comment]
The spelling mistake fix is still out :)

**Comment by gnachman** [comment]
Oops, thanks for the reminder.

**Pull request merged**